scripts/local.js

```diff
@@ -0,0 +1,60 @@
+const webdriver = require('selenium-webdriver');
+const { By } = require('selenium-webdriver');
+const assert = require('assert');
+var browserstack = require('browserstack-local');
+
+//creates an instance of Local
+var bs_local = new browserstack.Local();
+
+// replace <browserstack-accesskey> with your key. You can also set an environment variable - "BROWSERSTACK_ACCESS_KEY".
+var bs_local_args = { 'key': ACCESS_KEY };
+
+// starts the Local instance with the required arguments
+bs_local.start(bs_local_args, function() {
+  console.log("Started BrowserStackLocal");
+  // check if BrowserStack local instance is running
+  console.log(bs_local.isRunning());
+  runTestWithCaps(); 
+});
+// Input capabilities
+var capabilities = {
+	'bstack:options' : {
+		"os" : "OS X",
+		"osVersion" : "Sierra",
+		"buildName" : "Final-Snippet-Test",
+		"sessionName" : "Selenium-4 Nodejs snippet test",
+		"local" : "true",
+		"seleniumVersion" : "4.0.0",
+		"userName" : "USER_NAME",
+		"accessKey" : "ACCESS_KEY",
+	},
+	"browserName" : "Chrome",
+	"browserVersion" : "latest",
+}
+
+async function runTestWithCaps () {
+     let driver = new webdriver.Builder().usingServer(`https://rutvikchandla_2MEern:AXHzyg34Qr81Nep231pu@hub-cloud.browserstack.com/wd/hub`).withCapabilities(capabilities).build();
```

Comment:
Personal keys, please remove and also rotate your keys.
Plus we don't even need to pass creds as basic auth since we have it in caps.